src/components/hand-controls2.js

```diff
@@ -106,6 +106,7 @@ AFRAME.registerComponent("hand-controls2", {
       el.setAttribute("vive-controls", controlConfiguration);
```

Comment:
(why is this component called `hand-controls2`? Surely it deserves a better name)

Reply:
I think its a direct extension of the aframe built in component `hand-controls` . It's likely something we might want to land upstream but we don't know exactly what we want yet so its not worth maintaining a seperate fork of aframe over. Thus `hand-controls2` was born.